gammapy/utils/random/inverse_cdf.py

```diff
@@ -0,0 +1,79 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+import numpy as np
+from .utils import get_random_state
+
+
+class InverseCDFSampler:
+    """Inverse CDF sampler.
+       
+   It determines a set of random numbers and calculate the cumulative 
+   distribution function.
+   
+   Parameters
+   ----------
+   pdf : `~gammapy.maps.Map`
+        Map of the predicted source counts.
+   axis : integer
+        Axis along which sampling the indexes.
+   random_state : integer
```

Comment:
Please write this docstring like this:
https://docs.gammapy.org/0.12/development/howto.html#random-numbers

I use PyCharm and it uses docstring types to do type and code analysis, and if you write "int" here and later call a method ".uniform" on it it complains.

Reply:
Okay.